File: PYTHON/Kode/Pants.py
from Clothing import Clothing

class Pants(Clothing):
    def __init__(self, brand, size, price, fit_type):
        super().__init__(brand, size, price)
        self.fit_type = fit_type
    
    def get_fit_type(self): 
        return self.fit_type
    
    def set_fit_type(self, fit_type): 
        self.fit_type = fit_type
    
    def display(self):
        return super().display() + f" {self.fit_type:<13} |"

File: C++/Kode/Pants.cpp
// Pants.h
#ifndef PANTS_H
#define PANTS_H

#include "Clothing.cpp"

class Pants : public Clothing {
private:
    string fitType;

public:
    Pants(string brand, string size, double price, string fitType) : Clothing(brand, size, price) {
        this->fitType = fitType;
    }
    
    void setFitType(string fitType) { 
        this->fitType = fitType; 
    }

    string getFitType() { 
        return fitType; 
    }
    
    void display() override {
        Clothing::display();
        cout << " | " << setw(15) << fitType << " |" << endl;
    }
};

#endif